sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/catalog/ExternalCatalogUtils.scala

```diff
@@ -271,6 +271,14 @@ object CatalogUtils {
     }
   }
 
+  def makeQualifiedLocationPath(
```

Comment:
shall we rename this and the one above to `makeQualifiedDBObjectPath` as well?

Reply:
en.